src/connectors/bugs.js

```diff
@@ -16,7 +16,7 @@ Connector.durationSelector = '.finish';
 
 Connector.isPlaying = () => {
 	let btn = $('.btnPlay').first();
-	return btn.html() === '�Ͻ����'; // if text '�Ͻ����' in button, it means 'pause' because music is playing.
+	return btn.html() === '일시정지'; // if text 'ÀÏ½ÃÁ¤Áö' in button, it means 'pause' because music is playing.
```

Comment:
Can we remove the comment since I dont think it brings any value.

Reply:
Yes. you can remove that comment!